src/clj/witan/send/model/output.clj

```diff
@@ -85,8 +85,24 @@
                                    :beta (- (get-in population [calendar-year academic-year]) j)})))
                     coll years)) {} ages))
 
-(defn output-transitions [file projections]
-  (binding [*print-length* nil] (spit file (pr-str projections))))
+(defn output-transitions
```

Comment:
Do we need to do both of these outputs now? Also it seems a bit wrong using the same defn for a function which does two quite different things

Reply:
docstring would be cool too